docs/production/ingesters-with-wal.md

```diff
@@ -16,13 +16,13 @@ _The WAL is currently considered experimental._
 1. Since ingesters need to have the same persistent volume across restarts/rollout, all the ingesters should be run on [statefulset](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) with fixed volumes.
 
 2. Following flags needs to be set
-    * `--ingester.wal-dir` to the directory where the WAL data should be stores and/or recovered from. Note that this should be on the mounted volume.
     * `--ingester.wal-enabled` to `true` which enables writing to WAL during ingestion.
-    * `--ingester.checkpoint-enabled` to `true` to enable checkpointing of in-memory chunks to disk. This is optional which helps in speeding up the replay process.
+    * `--ingester.wal-dir` to the directory where the WAL data should be stores and/or recovered from. Note that this should be on the mounted volume.
     * `--ingester.checkpoint-duration` to the interval at which checkpoints should be created. Default is `30m`, and depending on the number of series, it can be brought down to `15m` if there are less series per ingester (say 1M).
     * `--ingester.recover-from-wal` to `true` to recover data from an existing WAL. The data is recovered even if WAL is disabled and this is set to `true`. The WAL dir needs to be set for this.
         * If you are going to enable WAL, it is advisable to always set this to `true`.
     * `--ingester.tokens-file-path` should be set to the filepath where the tokens should be stored. Note that this should be on the mounted volume. Why this is required is described below.
+    * `--ingester.checkpoint-enabled` to `true` to enable checkpointing of in-memory chunks to disk (`true` by default). There is no mechanism to delete the old WAL yet if checkpoint is disabled, so it should always be enabled unless for some (small) testing.
```

Comment:
Given now it's the default, can't we remove this list entry at all?

Reply:
Makes sense for the production doc.